src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/MakeBundleNativeCodeExternalTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.Build.Framework;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using Xamarin.Android.Tasks;
+
+namespace Xamarin.Android.Build.Tests {
+
+	[TestFixture]
+	public class MakeBundleNativeCodeExternalTests : BaseTest {
```

Comment:
I'd be happy to rename this class and file to something else considering that the 1 test in it is actually testing *both* the `MakeBundleNativeCodeExternal` and `Aot` tasks. I wasn't sure what would be the most sensible way to name the containing class and file for this test.